src/Integration/Resources/Strings.resx

```diff
@@ -438,6 +438,67 @@ This is the general format to use when writing errors to output window or when t
     <value>Warning: could not find CodeAnalysisRuleSet for project '{0}'</value>
     <comment>Warning message. '{0}' project unique name </comment>
   </data>
+  <data name="ConflictDetailWeakenedRulesTitle" xml:space="preserve">
+    <value>Weakened rules:</value>
+  </data>
+  <data name="RuleSetConflictsDetected" xml:space="preserve">
+    <value>The applied ruleset is weakening the SonarQube quality profile in one or more projects. See output window (General) for more details. You can also [fix]() that automatically.</value>
```

Comment:
The second sentence reads oddly. Possible alternative: "[Click here] to fix automatically."